util/Migrator/DbScripts/2023-11-14_00_2019FamilyPlanPremium.sql

```diff
@@ -4,3 +4,329 @@ SET
     [UsersGetPremium] = 0
 WHERE
     [PlanType] = 1 -- Families 2019 Annual
+    AND [Id] NOT IN ('dec8ffc3-24a8-4b5a-b057-ab7300c1ca1a',
+    'cc4b8cdd-1202-49de-8830-ab6201160e64',
+    '4dd79e1b-f687-431b-95b6-a9bf00bb9fa4',
+    'b285df2d-303a-4ca9-ae67-aa96014738d1',
```

Comment:
Once again, only being run once, but large in clauses can go sideways in sql plans, could consider joining:

Example:

-- Create a temporary table to hold the excluded IDs
CREATE TABLE #ExcludedOrgs
(
    ExcludedId UNIQUEIDENTIFIER
);

-- Insert the excluded IDs into the temporary table
INSERT INTO #ExcludedOrgs (ExcludedId)
VALUES
    ('dec8ffc3-24a8-4b5a-b057-ab7300c1ca1a'),
    ('cc4b8cdd-1202-49de-8830-ab6201160e64'),
    ('4dd79e1b-f687-431b-95b6-a9bf00bb9fa4'),
    ('b285df2d-303a-4ca9-ae67-aa96014738d1'),
    ('59f91ca0-648e-4cc9-a510-abf100ec1a21'),
    ('3ed5120e-83c2-4dcc-8b23-ab38014a0d91'),
    ('64dc181b-1d13-477f-8195-abd9012a26f4'),
    ('872ded84-c905-4b41-a2c0-ab340025b753'),
    ('3c711cd5-9637-405f-ae33-a9f90160ec48'),
    ('938521e7-ebf4-495a-843c-ac0b002af359');

-- Update the Organization table by performing a LEFT JOIN with the temporary table
UPDATE org
SET org.UsersGetPremium = 0
FROM [dbo].[Organization] org
LEFT JOIN #ExcludedOrgs ex
    ON org.Id = ex.ExcludedId
WHERE org.PlanType = 1 -- Families 2019 Annual
    AND ex.ExcludedId IS NULL;

-- Drop the temporary table if no longer needed
DROP TABLE #ExcludedOrgs;